app/Http/ViewComposers/MenuComposer.php

```diff
@@ -54,7 +55,8 @@ public function compose(View $view)
         /** @var User $user */
         $user = Auth::user();
 
-        $vars['navbar'] = in_array(Config::get('site_style'), ['mono', 'dark']) ? 'navbar-inverse' : '';
+        //Todo: should be handled via CSS Themes
+        $vars['navbar'] = in_array(UserPref::getPref($user, 'site_style'), ['mono', 'dark']) ? 'navbar-inverse' : '';
```

Comment:
This does not fall back to the global setting.

Reply:
missing fallback implemented